src/shared/form-validators/index.js

```diff
@@ -27,11 +27,16 @@ const mustMatch = (message, getPassword) => (value) => {
   const password = getPassword();
   return value === password || message;
 };
+const mustNotMatch = (message, getPassword) => (value) => {
+  const password = getPassword();
+  return value !== password || message;
+};
```

Comment:
We don't only use this function for the password but also use it for other cases, so the param `getPassword` is not good. It will be good if we can pass the string value or function to param `getPassword`

```suggestion
const mustNotMatch = (notMatchTo, errorMessage) => (value) => {
 const notMatchValue = typeof notMatchTo === 'function' ? notMatchTo() : notMatchTo;
  return value !== notMatchValue || message;
};
```